deploy-service/common/pom.xml

```diff
@@ -19,6 +19,26 @@
         <dropwizard.metrics.version>3.1.0</dropwizard.metrics.version>
     </properties>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <skip>false</skip>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
```

Comment:
what is it?

Reply:
Build a jar for test utils so they can be used by deploy-service. 
common-main -> deploy_service-main
New
common-test -> deploy_service-test